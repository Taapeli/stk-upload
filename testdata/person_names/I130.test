[description]
Latinalainen patronyymi

[options]

[original]
0 HEAD
1 CHAR UTF-8
0 @I130 INDI
1 NAME Johan Henrici/Mattila/
0 TRLR

[expected]
0 HEAD
1 CHAR UTF-8
0 @I130 INDI
1 NAME Johan/Mattila/Henrici
2 NOTE _orig_NAME Johan Henrici/Mattila/
2 NSFX Henrici
0 TRLR
