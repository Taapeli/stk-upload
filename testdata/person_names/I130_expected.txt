0 HEAD
1 CHAR UTF-8
0 @I130 INDI
1 NAME Johan/Mattila/Henrici2 NOTE _orig_NAME Johan Henrici/Mattila/
0 TRLR
1 CONT 
0 TRLR